src/core_modules/capture-core/reducers/descriptions/workingLists/workingLists.reducerDescription.js

```diff
@@ -2,8 +2,8 @@
 import { moment } from 'capture-core-utils/moment';
 import { createReducerDescription } from '../../../trackerRedux/trackerReducer';
 import {
-    actionTypes as columnSelectorActionTypes,
-} from '../../../components/ListView/ListWrapper/actions/columnSelectorDialog.actions';
+    columnSelectorActionTypes,
```

Comment:
👍 great stuff thanks for putting all the effort to make those according to what we have discussed. I am sure the effort will pay us off 🙇